BLS/README.md

```diff
@@ -0,0 +1,230 @@
+# BLS Module
+
+## Overview
+> Boneh–Lynn–Shacham (BLS) signature scheme on Barreto-Naehrig 254 bit curve (BN-254)
+
+BLS signature aggregation reduces the size of signature data to store on-chain
```

Comment:
it would be better if you add a reference link for BLS signature.
e.g. 
https://en.wikipedia.org/wiki/BLS_digital_signature